tests/expected_cmdline_output_files/istio-ingress_expl_output.txt

```diff
@@ -0,0 +1,949 @@
+ <text>
+  Configurations affecting the connectivity between (src)0.0.0.0/0 and (dst)default/reviews-v3(Deployment-StatefulSet):
+
+(dst)default/reviews-v3(Deployment-StatefulSet):
+Policy Configurations:
+Default-Policy
+Resource Configurations:
+default/reviews-v3(Deployment-StatefulSet): line 2596 in file ./istio_testcases/example_policies/istio-ingress-test/resources/bookinfo_pods_list.json
+ </text>
+</entry>
+<entry dst="prod/details-v1(Deployment-StatefulSet)" id="5" src="0.0.0.0/0">
+ <text>
+  Configurations affecting the connectivity between (src)0.0.0.0/0 and (dst)prod/details-v1(Deployment-StatefulSet):
+
+(dst)prod/details-v1(Deployment-StatefulSet):
+Policy Configurations:
+Default-Policy
+Resource Configurations:
+prod/details-v1(Deployment-StatefulSet): line 4 in file ./istio_testcases/example_policies/istio-ingress-test/resources/bookinfo_pods_list.json
+ </text>
+</entry>
+<entry dst="prod/ratings-v1(Deployment-StatefulSet)" id="6" src="0.0.0.0/0">
+ <text>
+  Configurations affecting the connectivity between (src)0.0.0.0/0 and (dst)prod/ratings-v1(Deployment-StatefulSet):
+
+(dst)prod/ratings-v1(Deployment-StatefulSet):
+Policy Configurations:
+Default-Policy
+Resource Configurations:
+prod/ratings-v1(Deployment-StatefulSet): line 1028 in file ./istio_testcases/example_policies/istio-ingress-test/resources/bookinfo_pods_list.json
+ </text>
+</entry>
+<entry dst="qa/reviews-v1(Deployment-StatefulSet)" id="7" src="0.0.0.0/0">
+ <text>
+  Configurations affecting the connectivity between (src)0.0.0.0/0 and (dst)qa/reviews-v1(Deployment-StatefulSet):
+
+(dst)qa/reviews-v1(Deployment-StatefulSet):
+Policy Configurations:
+Default-Policy
+Resource Configurations:
+qa/reviews-v1(Deployment-StatefulSet): line 1536 in file ./istio_testcases/example_policies/istio-ingress-test/resources/bookinfo_pods_list.json
+ </text>
+</entry>
+<entry dst="qa/reviews-v2(Deployment-StatefulSet)" id="8" src="0.0.0.0/0">
+ <text>
+  Configurations affecting the connectivity between (src)0.0.0.0/0 and (dst)qa/reviews-v2(Deployment-StatefulSet):
+
+(dst)qa/reviews-v2(Deployment-StatefulSet):
+Policy Configurations:
+Default-Policy
+Resource Configurations:
+qa/reviews-v2(Deployment-StatefulSet): line 2066 in file ./istio_testcases/example_policies/istio-ingress-test/resources/bookinfo_pods_list.json
+ </text>
+</entry>
+<entry dst="0.0.0.0/0" id="9" src="default/istio-ingressgateway(Deployment)">
+ <text>
+  Configurations affecting the LACK of connectivity between (src)default/istio-ingressgateway(Deployment) and (dst)0.0.0.0/0:
+
+(src)default/istio-ingressgateway(Deployment):
+Policy Configurations:
+Default-Policy
+None: line 1 in file ./istio_testcases/example_policies/istio-ingress-test/resources/virtual_services.yaml
+Resource Configurations:
+default/istio-ingressgateway(Deployment): line 1 in file ./istio_testcases/example_policies/istio-ingress-test/resources/istio-ingress-gateway-deployment.yaml
```

Comment:
why is the policy configuration named `None`?

Reply:
Fixed.

Reply:
Now `bookinfo-rule/eu.bookinfo.com, uk.bookinfo.com, reviews.prod.svc.cluster.local/allow:` is the policy name?